src/js/helpers/moneyFormatter.js

```diff
@@ -186,6 +186,39 @@ export const formatTreemapValues = (value) => {
     return `${formattedCurrency}${longLabel}`;
 };
 
+export const formatStateValues = (value) => {
```

Comment:
Can we move this logic to the model? moneyFormatter should be more generic helpers. Since this is specifically for Top 5 data, it can go in the getter function for `amount` in BaseStateCategoryResult.js. Or, if we format money in the same way (conditionally, based on amount) throughout the site, we can give this a more generic name for re-use.